### ISSUE TITLE
EDSC-997: ESI orders stuck in Submitting state in SIT

### ISSUE THREAD

**Issue by earthdata-tasktop** [nasa/edsc-search-test-config-3#439]
From [~abdulgkhan]:  
I am trying to submit a service request to EDSC.  
I query the granules using [https://search.sit.earthdata.nasa.gov/search/granules?p=C1000000672-DEV07&amp;tl=1442522369!4!!&amp;q=MOD14A1+DEV07](https://search.sit.earthdata.nasa.gov/search/granules?p=C1000000672-DEV07&tl=1442522369!4!!&q=MOD14A1+DEV07), pick the first one on the list then reproject to UTM and pick a “zion” and a “ZLAT”

I am presented with the following screen, but never see the request show up in DEV07 and never get an EMAIL.

I just submitted one, but also submitted one on Friday before the weekend.

From [~bo.sun]:  
We were able to reproduce the issue in SIT. The ESI request failed in our background jobs. We couldn’t see the log because of EI-4682([https://bugs.earthdata.nasa.gov/browse/EI-4682](https://bugs.earthdata.nasa.gov/browse/EI-4682)). Unfortunately, our hands are tied until the ticket is resolved… Will let you know when we have some updates/progress!

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

I asked Abdul Khan, the original reporter, to retest in SIT after the firewall issue was resolved.

Email exchange with Abdul:  

Hi Abdul:  Thanks very much for re-testing this SIT issue - much appreciated.

I want to just be sure I understand your results:

1. You said "An AST_L1T service request for all FRB images went through. "  

   Does this mean that we can Verify EDSC-997 as fixed?

Yes

1. You reported two other problems:  

   a. the “Additional Resources and Documentation” Link gives a page not found.  

   b. when I clicked the button “View Browse Image Links”, the HTML page was blank.

Do we need to open separate tickets for those two problems, or do those problems mean that EDSC-997 cannot be verified at this time?

I think these are separate issues.  I know that 2a is because that happens in production.  I am holding out hope that 2b was a network issue between EDSC and CMR or something that can not be reproduced.

(NOTE:  Per Patrick:  

The page not found is a case of garbage in / garbage out. The link is coming straight from the metadata.  

The browse one would require further investigation, but it could very well be a metadata issue as well (a collection that used to have browse imagery links and no longer does, for instance).)

**Comment by earthdata-tasktop** [comment]
(Comment from Patrick Quinn):

Approved in 3/28 sprint planning

**Comment by earthdata-tasktop** [comment]
(Comment from Dana Shum):

Migrating "Internal Verified" tickets to "Completed".

**Comment by earthdata-tasktop** [comment]
(Comment from Bo Sun):

TTPro Ticket #5004943 for opening up the firewall port.

**Comment by earthdata-tasktop** [comment]
(Comment from Bo Sun):

The issue is fixed with the port(s) opened in the firewall during the PM.

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

Sent an email to Abdul Khan asking him to retest in SIT.

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

I asked Abdul Khan, the original reporter, to retest in SIT after the firewall issue was resolved.

Email exchange with Abdul:  

Hi Abdul:  Thanks very much for re-testing this SIT issue - much appreciated.

I want to just be sure I understand your results:

1. You said "An AST_L1T service request for all FRB images went through. "  

   Does this mean that we can Verify EDSC-997 as fixed?

Yes

1. You reported two other problems:  

   a. the “Additional Resources and Documentation” Link gives a page not found.  

   b. when I clicked the button “View Browse Image Links”, the HTML page was blank.

Do we need to open separate tickets for those two problems, or do those problems mean that EDSC-997 cannot be verified at this time?

I think these are separate issues.  I know that 2a is because that happens in production.  I am holding out hope that 2b was a network issue between EDSC and CMR or something that can not be reproduced.

(NOTE:  Per Patrick:  

The page not found is a case of garbage in / garbage out. The link is coming straight from the metadata.  

The browse one would require further investigation, but it could very well be a metadata issue as well (a collection that used to have browse imagery links and no longer does, for instance).)

**Comment by earthdata-tasktop** [comment]
(Comment from Patrick Quinn):

Approved in 3/28 sprint planning

**Comment by earthdata-tasktop** [comment]
(Comment from Bo Sun):

TTPro Ticket #5004943 for opening up the firewall port.

**Comment by earthdata-tasktop** [comment]
(Comment from Bo Sun):

The issue is fixed with the port(s) opened in the firewall during the PM.

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

Sent an email to Abdul Khan asking him to retest in SIT.

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

I asked Abdul Khan, the original reporter, to retest in SIT after the firewall issue was resolved.

Email exchange with Abdul:  

Hi Abdul:  Thanks very much for re-testing this SIT issue - much appreciated.

I want to just be sure I understand your results:

1. You said "An AST_L1T service request for all FRB images went through. "  

   Does this mean that we can Verify EDSC-997 as fixed?

Yes

1. You reported two other problems:  

   a. the “Additional Resources and Documentation” Link gives a page not found.  

   b. when I clicked the button “View Browse Image Links”, the HTML page was blank.

Do we need to open separate tickets for those two problems, or do those problems mean that EDSC-997 cannot be verified at this time?

I think these are separate issues.  I know that 2a is because that happens in production.  I am holding out hope that 2b was a network issue between EDSC and CMR or something that can not be reproduced.

(NOTE:  Per Patrick:  

The page not found is a case of garbage in / garbage out. The link is coming straight from the metadata.  

The browse one would require further investigation, but it could very well be a metadata issue as well (a collection that used to have browse imagery links and no longer does, for instance).)

**Comment by earthdata-tasktop** [comment]
(Comment from Patrick Quinn):

Approved in 3/28 sprint planning

**Comment by earthdata-tasktop** [comment]
(Comment from Bo Sun):

The issue is fixed with the port(s) opened in the firewall during the PM.

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

Sent an email to Abdul Khan asking him to retest in SIT.

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

I asked Abdul Khan, the original reporter, to retest in SIT after the firewall issue was resolved.

Email exchange with Abdul:  

Hi Abdul:  Thanks very much for re-testing this SIT issue - much appreciated.

I want to just be sure I understand your results:

1. You said "An AST_L1T service request for all FRB images went through. "  

   Does this mean that we can Verify EDSC-997 as fixed?

Yes

1. You reported two other problems:  

   a. the “Additional Resources and Documentation” Link gives a page not found.  

   b. when I clicked the button “View Browse Image Links”, the HTML page was blank.

Do we need to open separate tickets for those two problems, or do those problems mean that EDSC-997 cannot be verified at this time?

I think these are separate issues.  I know that 2a is because that happens in production.  I am holding out hope that 2b was a network issue between EDSC and CMR or something that can not be reproduced.

(NOTE:  Per Patrick:  

The page not found is a case of garbage in / garbage out. The link is coming straight from the metadata.  

The browse one would require further investigation, but it could very well be a metadata issue as well (a collection that used to have browse imagery links and no longer does, for instance).)

**Comment by earthdata-tasktop** [comment]
(Comment from Patrick Quinn):

Approved in 3/28 sprint planning

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

Sent an email to Abdul Khan asking him to retest in SIT.

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

I asked Abdul Khan, the original reporter, to retest in SIT after the firewall issue was resolved.

Email exchange with Abdul:  

Hi Abdul:  Thanks very much for re-testing this SIT issue - much appreciated.

I want to just be sure I understand your results:

1. You said "An AST_L1T service request for all FRB images went through. "  

   Does this mean that we can Verify EDSC-997 as fixed?

Yes

1. You reported two other problems:  

   a. the “Additional Resources and Documentation” Link gives a page not found.  

   b. when I clicked the button “View Browse Image Links”, the HTML page was blank.

Do we need to open separate tickets for those two problems, or do those problems mean that EDSC-997 cannot be verified at this time?

I think these are separate issues.  I know that 2a is because that happens in production.  I am holding out hope that 2b was a network issue between EDSC and CMR or something that can not be reproduced.

(NOTE:  Per Patrick:  

The page not found is a case of garbage in / garbage out. The link is coming straight from the metadata.  

The browse one would require further investigation, but it could very well be a metadata issue as well (a collection that used to have browse imagery links and no longer does, for instance).)

**Comment by earthdata-tasktop** [comment]
(Comment from Patrick Quinn):

Approved in 3/28 sprint planning

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

I asked Abdul Khan, the original reporter, to retest in SIT after the firewall issue was resolved.

Email exchange with Abdul:  

Hi Abdul:  Thanks very much for re-testing this SIT issue - much appreciated.

I want to just be sure I understand your results:

1. You said "An AST_L1T service request for all FRB images went through. "  

   Does this mean that we can Verify EDSC-997 as fixed?

Yes

1. You reported two other problems:  

   a. the “Additional Resources and Documentation” Link gives a page not found.  

   b. when I clicked the button “View Browse Image Links”, the HTML page was blank.

Do we need to open separate tickets for those two problems, or do those problems mean that EDSC-997 cannot be verified at this time?

I think these are separate issues.  I know that 2a is because that happens in production.  I am holding out hope that 2b was a network issue between EDSC and CMR or something that can not be reproduced.

(NOTE:  Per Patrick:  

The page not found is a case of garbage in / garbage out. The link is coming straight from the metadata.  

The browse one would require further investigation, but it could very well be a metadata issue as well (a collection that used to have browse imagery links and no longer does, for instance).)
